frontend/pages/index.tsx

```diff
@@ -22,6 +46,19 @@ const HomePage = () => {
       <h2>Welcome to Republic of Rome Online</h2>
       <p><i>Experience the intrigue and power struggles of Ancient Rome, right from your browser</i></p>
 
+      <h3 id="waitlist">Join our waitlist</h3>
+        <form onSubmit={handleSubmit}>
+          <Stack alignItems="start" spacing={2}>
+            <TextField required
+                id="email"
+                label="Email"
+                onChange={handleEmailChange}
+                helperText={"johndoe@example.com"}
+                style={{width: "300px"}} />
```

Comment:
It would be good to use the `error` attribute to show a feedback message if there is any. This message will be captured by the `requestWithoutAuthentication` function already, you just need to extract it from `response` in a similar manner to frontend/pages/games/new.tsx.

Reply:
This was very interesting. Thank you for pointing me in the right direction.